### PR TITLE
Fix the broken style on cluster/centos

### PR DESCRIPTION
**Pull request by xiaoping378**
kubeadm has some issues for me ...
I think the script style is more flexible, so please merge this PR.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33818)

### CODE REVIEW

File: cluster/centos/config-default.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ export FLANNEL_NET=${FLANNEL_NET:-"172.16.0.0/16"}
 
 # Admission Controllers to invoke prior to persisting objects in cluster
 # If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
-export ADMISSION_CONTROL=NamespaceLifecycle,NamespaceExists,LimitRanger,ServiceAccount,SecurityContextDeny,DefaultStorageClass,ResourceQuota
+export ADMISSION_CONTROL=NamespaceLifecycle,LimitRanger,ServiceAccount,ResourceQuota
```
Comment (xiaoping378) [inline cluster/centos/config-default.sh]: refer to [kubernetes docs](http://kubernetes.io/docs/admin/admission-controllers/), the bottom of Documentation.

File: cluster/centos/node/scripts/docker.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ DOCKER_OPTS=${1:-""}
 DOCKER_CONFIG=/opt/kubernetes/cfg/docker
 
 cat <<EOF >$DOCKER_CONFIG
-DOCKER_OPTS="-H tcp://127.0.0.1:4243 -H unix:///var/run/docker.sock -s devicemapper --selinux-enabled=false ${DOCKER_OPTS}"
+DOCKER_OPTS="-H tcp://127.0.0.1:4243 -H unix:///var/run/docker.sock -s overlay --selinux-enabled=false ${DOCKER_OPTS}"
```
Comment (xiaoping378) [inline cluster/centos/node/scripts/docker.sh]: if the centos7 is minimal,  it will be failed due to lack of dependency

**Comment by errordeveloper** [comment]
> kubeadm has some issues for me ...

Could you please provide some details of what are those issue exactly?

> I think the script style is more flexible

I am afraid it's been deprecated as it's very difficult to maintain upstream. We are looking to delete most of the code in `cluster` directory in very near future, surely you are still welcome to fork it in any way.

**Comment by roberthbailey** [comment]
@xiaoping378 sorry for the delay (I had a sick day on Friday). Could you start by squashing your commits?

**Comment by xiaoping378** [comment]
Hi @roberthbailey , i have checked another PR #34002

@errordeveloper, the main issue i met is the network problems --- downloading dependents timeout.

**Comment by roberthbailey** [comment]
Closing in favor of #34002. For future reference, you don't have to open a new PR to squash your commits. You just need to squash them locally and then force push to your fork.

**Pull request closed without merging**